### PR TITLE
Restaurant feature

### PR DESCRIPTION
**Pull request by Byabasaija**
In this pull request,
I built a simple javascript restaurant page with navigation tabs that display specific content when clicked without refreshing the page. I added simple styling to make the page look nice.
I used JS modules to write my code in an organized way. 
**Usage**
- open the project in the browser
- click on different tabs display relevant content

Feedback about this project is highly appreciated.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (MuriloRoque) [inline .vscode/settings.json:3]: - [x] This file shouldn't be committed, please add it to `.gitignore`.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "eslint-config-airbnb-base": "14.1.x",
+    "eslint-plugin-import": "2.20.x",
+    "stylelint": "13.3.x",
+    "stylelint-config-standard": "20.0.x",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "3.17.x",
+    "webpack": "^5.26.2",
+    "webpack-cli": "^4.5.0"
+  }
+}
```
Comment (MuriloRoque) [inline package.json]: - [x] You are missing some scripts keys here with the commands to run your application with Webpack, here's a basic example:

```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @Byabasaija :wave: 

### You have done an amazing job on this project, your page is working as expected, and your code is DRY, including your professional repository, very nice! :clap: :clap: :clap: 

### There are only some details related to your scripts and ignored files, please check my comments in your code below :smile: 

Kindly make the requested changes and submit them for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @Byabasaija 👋

**You've done pretty well here fixing a few issues raised in the previous review. 👌**

However, let's make a couple of changes.

- [x] Please deploy the project and include the live link in the project readme.

- [x] In the getting started section of the readme, please include instructions to clone the project, run npm install, and run the npm start script.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @Byabasaija :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**